index.js

```diff
@@ -178,9 +199,20 @@ function getCarInfoById(/* code here */) {
  *     (1) an array which is an inventory of cars like the one inside /data/inventory.js.
  * sortCarInventory returns an inventory that is sorted by car_model, ascending [A-Z].
 */
-function sortCarInventory(/* code here */) {
-  /* code here */
-}
+function sortCarInventory(inventory) {
+  return inventory.sort(function (model1,model2){
+    let a = model1.car_model.toUpperCase();
+    let b = model2.car_model.toUpperCase();
+
+    if(a<b) {
+      return -1;
+    }
+    if (a>b) {
+      return 1;
+    }
+    return 0;
+  })}
+
```

Comment:
This function could be consolidated into a couple of lines like this: 

`function sortCarInventory(inventory) {
  return inventory.sort((a, b) => (a.car_model > b.car_model ? 1 : -1));
}`